src/main/java/com/makefire/anonymous/domain/user/entity/Board.java

```diff
@@ -0,0 +1,40 @@
+package com.makefire.anonymous.domain.user.entity;
+
+import lombok.*;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+/**
+ * packageName : com.makefire.anonymous
+ * fileName : Board
+ * author : 강정호
+ * date : 22-02-22
+ * description : 게시판 엔티티
+ * =================================
+ * DATE     AUTHOR   NOTE
+ * 22-02-22 강정호
+ * ---------------------------------
+ */
+@Getter
+@Entity
+@NoArgsConstructor(access = AccessLevel.PUBLIC)
```

Comment:
Builder 패턴 가져가는 방식 굳 😀
다만 접근 범위를 `PUBLIC`에서 좁게 가져가는건 어떨지..!

```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```

Reply:
🦥끝났습니당